components/event-simulator/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/internal/CarbonEventSimulator.java

```diff
@@ -551,19 +552,26 @@ public void run() {
                 File file = new File(path);
                 FileInputStream fis = null;
                 BufferedInputStream bis = null;
-                DataInputStream dis = null;
+                BufferedReader br = null;
```

Comment:
Would have been better to select descriptive variable name rather than "br". Also better to do the same for "fis", and "bis"